src/applications/appeals/996/content/introduction.jsx

```diff
@@ -0,0 +1,256 @@
+              You can ask us to review more than 1 issue.
+            </li>
+            <li>Your contact information</li>
+          </ul>
+        </va-process-list-item>
+        <va-process-list-item header="Start your request">
+          <p>
+            We’ll take you through each step of the process. It takes about 15
+            minutes.
+          </p>
+          <p>
+            If you need help requesting a Higher-Level Review, you can appoint
+            an accredited representative or Veterans Service Organization (VSO)
+            to work with us on your behalf.
+          </p>
+          <p>
+            <a href={GET_HELP_REP_OR_VSO_URL}>
+              Get help requesting a Higher-Level Review
+            </a>
+          </p>
+          <va-additional-info trigger="What happens after you submit your request?">
+            <div>
+              You don’t need to do anything unless we send you a letter asking
+              for more information. If we schedule any exams for you, be sure
+              not to miss them.
+            </div>
+          </va-additional-info>
+        </va-process-list-item>
+      </va-process-list>
+    </Toggler.Enabled>
+    <Toggler.Disabled>
+      <va-process-list uswds>
+        <va-process-list-item header="Prepare">
+          <p>To fill out this application, you’ll need your:</p>
+          <ul>
+            <li>Primary address</li>
+            <li>
+              List of issues you disagree with and the VA decision date for each
+            </li>
+            <li>Representative’s contact information (optional)</li>
+          </ul>
+          <p>
+            <strong>What if I need help with my application?</strong>
+          </p>
+          <p>
+            If you need help requesting a Higher-Level Review, you can contact a
+            VA regional office and ask to speak to a representative. To find the
+            nearest regional office, please call{' '}
+            <va-telephone contact={CONTACTS.VA_BENEFITS} />
+            {' or '}
+            <a href={FACILITY_LOCATOR_URL}>visit our facility locator tool</a>.
+          </p>
+          <p>
+            A Veterans Service Organization or VA-accredited attorney or agent
+            can also help you request a decision review.
+          </p>
+          <a href={GET_HELP_REVIEW_REQUEST_URL}>
+            Get help requesting a decision review
+          </a>
+          .
+        </va-process-list-item>
+        <va-process-list-item header="Start your request">
+          <p>
+            Complete this Higher-Level Review form. After submitting the form,
+            you’ll get a confirmation message. You can print this for your
+            records.
+          </p>
+        </va-process-list-item>
+      </va-process-list>
+    </Toggler.Disabled>
+  </Toggler>
+);
+
+export const processListTitle = 'Follow the steps to get started';
+export const processList = (
```

Comment:
is this used anywhere? at first glance, it looks similar to what's in the `<Toggler.Enabled>` above

Reply:
Ahhh, you're right. I just removed the duplicates.